### ISSUE TITLE
VirtualBox

### ISSUE THREAD

**Issue by danboid** [haikuports/haikuports#6497]
I see qemu has been already been ported but does Haiku support hardware accelerated virtualisation? I don't think recent versions of VB will run without it.

**Comment by pulkomandy** [comment]
No, there is currently no virtualization support. Qemu runs in emulation mode and it is slow and cpu hungry as aresult.

**Comment by bugcrazy** [comment]
There is this link on Haiku's Gsoc page about the initial port in 2010, there are only 2 patches in diff:

https://web.archive.org/web/20170425051113/http://article.gmane.org/gmane.comp.emulators.virtualbox.devel/3384
